packages/styles/src/utils.ts

```diff
@@ -130,3 +130,14 @@ export const getThemesVars = (themes: HvThemeStructure[]) => {
 
   return vars;
 };
+
+export const getVarValue = (cssVar: string, rootElementId?: string) => {
+  const root = document.getElementById(rootElementId || "hv-root");
```

Comment:
It's a bit strange the `styles` package is manipulating/accessing the DOM.
Also, isn't this function a "hack" needed just in some core components?

Reply:
Yeah, you're right, I'll revert that.